server/chat-plugins/daily-spotlight.ts

```diff
@@ -72,7 +72,7 @@ export const pages: PageTable = {
 
 export const commands: ChatCommands = {
 	removedaily(target, room, user) {
-		if (!room.chatRoomData) return this.errorReply("This command is unavailable in temporary rooms.");
+		if (!room.settings) return this.errorReply("This command is unavailable in temporary rooms.");
```

Comment:
```suggestion
		if (!room.settings.persistSettings) return this.errorReply("This command is unavailable in temporary rooms.");
```

...didn't you say you've fixed all these already?